lib/ui/dimmer/dimmer.js

```diff
@@ -0,0 +1,41 @@
+// Original => http://bootsnipp.com/snippets/78VV
+(function(root) {
+
+  'use strict';
+
+  var availity = root.availity;
+
+  availity.ui.provider('avDimmerConfig', function() {
+
+    var config = {
```

Comment:
avDimmerConfig should have all defaulting values. You should be able to configure the animation properties, classes, and events used for dimming.

```
var config = {
  duration: 250,
  showAnimation: 'fadeIn',
  showEvent: 'mouseenter',
  hideAnimation: 'fadeOut',
  hideEvent: 'mouseexit',
  overlaySelector: '.card-img-overlay'
};
```